.github/scripts/change-workflow-instance-states.py

```diff
@@ -1,13 +1,12 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # Changes the state of instances associated with the CI run's unique tag.
 # Where a run is a workflow in CircleCI. Can be used to start, stop, or
```

Comment:
Can you do a pass through the comments and remove / update references to CircleCI as needed